src/features/game/events/landExpansion/fertiliseCrop.ts

```diff
@@ -19,25 +19,28 @@ type Options = {
 };
 
 export enum FERTILISE_CROP_ERRORS {
-  EMPTY_EXPANSION = "Expansion does not exist!",
-  EXPANSION_NO_PLOTS = "Expansion does not have any plots!",
   EMPTY_PLOT = "Plot does not exist!",
   EMPTY_CROP = "There is no crop planted!",
   READY_TO_HARVEST = "Crop is ready to harvest!",
   CROP_ALREADY_FERTILISED = "Crop is already fertilised!",
   NO_FERTILISER_SELECTED = "No fertiliser selected!",
   NOT_A_FERTILISER = "Not a fertiliser!",
   NOT_ENOUGH_FERTILISER = "Not enough fertiliser!",
-  INVALID_PLANT = "Invalid plant!",
 }
 
-// Seeds which are implemented
-const VALID_FERTILISERS: InventoryItemName[] = ["Rapid Growth"];
-
-const isFertiliser = (
-  fertiliser: InventoryItemName
-): fertiliser is FertiliserName => {
-  return VALID_FERTILISERS.includes(fertiliser);
+const getPlantedAt = (
+  fertiliser: CropCompostName,
+  plantedAt: number,
+  fertilisedAt: number,
+  cropDetails: Crop
+) => {
+  const timeToHarvest = cropDetails.harvestSeconds * 1000;
+  const harvestTime = plantedAt + timeToHarvest;
+  const timeReduction = (harvestTime - fertilisedAt) / 2;
```

Comment:
Interesting.  So you can "late fertilize" to ("inefficiently use" fertilizer to) speed up a crop.